packages/babylon/src/tokenizer/index.js

```diff
@@ -445,7 +446,7 @@ export default class Tokenizer extends LocationParser {
       type = tt.exponent;
     }
 
-    if (next === 61) {
+    if (next === 61 && !exprAllowed) {
```

Comment:
(This has nothing to do with your PR)

The char codes are very difficult to read to me since I need to lookup in the table. It's sometimes inconsistent between int or hex representation.

I would like to PR to add a constant file where the char codes are stored and replace every occurrence in Babylon. What do you think?

Reply:
This seems to be a good idea which will make the code readable. I'll make a separate PR.This seems to be a good idea which will make the code readable. I'll make a separate PR.

Reply:
I would wait for more thoughts about since, especially @danez which is the main contrib of the parse.

Actually I already started locally. I can push a PR today, we can also both work on it.

Reply:
good idea